cbt-bot/cbt_bot_proto/cbt_bot_llm.proto

```
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package cbt_bot;
```

Buf (1.47.2)

Files with package "cbt_bot" must be within a directory "cbt_bot" relative to root but were in directory "cbt-bot/cbt_bot_proto".

(PACKAGE_DIRECTORY_MATCH)